scripts/jenkins/Jenkinsfile

```diff
@@ -477,7 +478,7 @@ def generateMainBuildStages() {
       CMAKE_FLAGS_DEBUG +
       CMAKE_FLAGS_BUILD_FULL +
       CMAKE_FLAGS_BUILD_STATIC + [
-      'PLUGINS': 'ALL;-crypto',
+      'PLUGINS': 'ALL',
```

Comment:
No need to change this now but just fyi: CMAKE_FLAGS_BUILD_ALL already contains `'PLUGINS': 'ALL'` among other things, so you only need to provide this line if you want to override the default.

I'll clean it up in a different PR, priority is to merge this sooner and get more stable builds.